tests/h/routes_test.py

```diff
@@ -90,6 +90,7 @@ def test_includeme():
              '/api/annotations/{id:[A-Za-z0-9_-]{20,22}}.jsonld',
              factory='h.traversal:AnnotationRoot',
              traverse='/{id}'),
+        call('api.groups', '/api/groups'),
```

Comment:
TIL: Apparently routes have to be in the same order in the routes test as they are in the routes module :)